src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -609,6 +610,8 @@ private void updateAccountAndRoles(App app, Account account, StudyParticipant pa
             account.setNote(participant.getNote());
         }
         account.setMigrationVersion(MIGRATION_VERSION);
+        // TODO: check if this needs to retain previous value if it's getting a null from participant
+        account.setClientTimeZone(participant.getClientTimeZone());
```

Comment:
From all discussions, this is a field that can be set by the participant or a researchers, it can be added/changed/deleted, so no special anything required for it.

Reply:
Ah! You are right. So far we've stored this value as a string, and just validated that it can be converted to a valid DateTimeZone object in the validators. And you could do this here...the key thing is that in the database column, the value should be the region string, such as "America/Los_Angeles".

Reply:
When you get to testing JSON serialization, you may find it's easier to store it as a string...

Reply:
I see what you mean. I'll definitely switch it to a string and just validate it.